### DISCUSSION TITLE
Joinfaces 5.0.x SNAPSHOT availability in maven repository

### DISCUSSION THREAD

**Discussion by suyambutriveni** [joinfaces/joinfaces#1352] [Q&A] [answered]
Hi Team,

I am looking forward to use the version 5.0.x, which comes with Primefaces 11 and spring boot 3 snapshot. 
is the 5.0.x-SNAPSHOT version available in maven repository?

Thank you
Ganesh

**Comment by larsgrefer** [comment, marked as answer]
Joinfaces SNAPSHOTs are available here: https://oss.sonatype.org/content/repositories/snapshots

**Reply by suyambutriveni** [reply]
Thank you. But Primefaces spring boot starter is not pulling the Primefaces 11 dependency, the issue is because of the classifier "jakarta" in the file https://github.com/joinfaces/joinfaces/blob/5.0.x/joinfaces-starters/primefaces-spring-boot-starter/build.gradle

There is no dependency with that classifier in the maven central.

**Reply by suyambutriveni** [reply]
The spring boot starter pulls the Primefaces 11 dependency if the classifier "jakarta" is removed from the gradle file.

**Reply by persapiens** [reply]
@suyambuganesh82 it is fixed in #1360 1360
